OAuth/ResourceOwner/GenericOAuth2ResourceOwner.php

```diff
@@ -211,6 +220,7 @@ protected function configureOptions(OptionsResolver $resolver)
             'attr_name' => 'access_token',
             'use_commas_in_scope' => false,
             'use_bearer_authorization' => true,
+            'get_token_with_basic_auth' => true,
```

Comment:
Just let's be consistent with existing options: `use_authorization_to_get_token`, and we can merge this :)

Reply:
Done! :+1: